From 86710c2d4b46680c02301765dd79e465819c8f19 Mon Sep 17 00:00:00 2001
From: Bernhard Kerbl <kerbl@icg.tugraz.at>
Date: Thu, 18 Jul 2024 08:08:44 +0200
Subject: [PATCH] Compatibility

Source: https://gitlab.inria.fr/bkerbl/simple-knn/-/commit/86710c2d4b46680c02301765dd79e465819c8f19
Applied only to the isolated build copy of upstream 44f7642. The frozen
official training source worktree remains unchanged.
---
 simple_knn.cu | 1 +
 1 file changed, 1 insertion(+)

diff --git a/simple_knn.cu b/simple_knn.cu
index e72e4c9..d7f8559 100644
--- a/simple_knn.cu
+++ b/simple_knn.cu
@@ -14,6 +14,7 @@
 #include "cuda_runtime.h"
 #include "device_launch_parameters.h"
 #include "simple_knn.h"
+#include <cfloat>
 #include <cub/cub.cuh>
 #include <cub/device/device_radix_sort.cuh>
 #include <vector>
